### ISSUE TITLE
Intellisense fails if a path to an image cannot be resolved

### ISSUE THREAD

**Issue by bjarketrux** [wkillerud/some-sass#182]
Hover intellisense and navigations fails with the following message:

```
[Error - 10:15:41 AM] Request textDocument/hover failed.
  Message: Request textDocument/hover failed with message: ENOENT: no such file or directory, realpath 'c:/my-work-dir/path/image.svg'
  Code: -32603 
```

The css is a simple background image:

```css
div {
  background-image: url(/path/image.svg);
}
```

I'm using Angular so the path is absolute to the final server path and cannot be combined directly with the working directory.

- VS Code version: v1.91.1
- Extension version: v3.1.7
- Operating system: Windows

Reproducible Case:

Steps to Reproduce:
1. Add an invalid `background-image: url(/does-not-exit);` to the scss
2. Hover over a mixin imported from a scss file (using `@use`).

Observe: No tooltip or navigation.
Expected: Tooltip shown (even if images are invalid and cannot be loaded).

**Comment by wkillerud** [comment]
Hello, thank you for reporting this issue. I've reproduced it on my end and identified the cause.

**Comment by wkillerud** [comment]
Fixed in [3.2.8](https://github.com/wkillerud/some-sass/releases/tag/some-sass%403.1.8)

**Comment by bjarketrux** [comment]
I can confirm that it has been fixed. Thank you for the swift update 👍
